### ISSUE TITLE
NotificareUtils.loadBitmap.onLoadFailed Crash

### ISSUE THREAD

**Issue by ali-star** [Notificare/notificare-sdk-android#66]
We're having this issue with Notificare that's happening to some users.

Notificare version:
`3.5.0`

Package:
`re.notifica.internal`

File:
`NotificareUtils.kt:86`

Stacktrace:
```
Fatal Exception: com.bumptech.glide.load.engine.CallbackException: Unexpected exception thrown by non-Glide code
       at com.bumptech.glide.load.engine.EngineJob.callCallbackOnLoadFailed(EngineJob.java:175)
       at com.bumptech.glide.load.engine.EngineJob$CallLoadFailed.run(EngineJob.java:402)
       at android.os.Handler.handleCallback(Handler.java:808)
       at android.os.Handler.dispatchMessage(Handler.java:101)
       at android.os.Looper.loop(Looper.java:166)
       at android.app.ActivityThread.main(ActivityThread.java:7529)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
Caused by java.lang.IllegalStateException: Already resumed, but proposed with update CompletedExceptionally[java.lang.RuntimeException: Failed to load the bit at https://push.notifica.re/upload/notification/7b906301041a3690b3a8046f03c0d6fe8a901c04b6a30c9bb6adca1c3eb46789/2f358d0cfce310f51e20369dd4acf5ecfd3ad5d1db90bae61a710548d5e25c71]
       at kotlinx.coroutines.CancellableContinuationImpl.alreadyResumedError(CancellableContinuationImpl.kt:482)
       at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl(CancellableContinuationImpl.kt:447)
       at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$default(CancellableContinuationImpl.kt:420)
       at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:328)
       at re.notifica.internal.NotificareUtils$loadBitmap$2$1.onLoadFailed(NotificareUtils.kt:86)
       at com.bumptech.glide.request.SingleRequest.setErrorPlaceholder(SingleRequest.java:444)
       at com.bumptech.glide.request.SingleRequest.onLoadFailed(SingleRequest.java:715)
       at com.bumptech.glide.request.SingleRequest.onLoadFailed(SingleRequest.java:671)
       at com.bumptech.glide.load.engine.EngineJob.callCallbackOnLoadFailed(EngineJob.java:173)
       at com.bumptech.glide.load.engine.EngineJob$CallLoadFailed.run(EngineJob.java:402)
       at android.os.Handler.handleCallback(Handler.java:808)
       at android.os.Handler.dispatchMessage(Handler.java:101)
       at android.os.Looper.loop(Looper.java:166)
       at android.app.ActivityThread.main(ActivityThread.java:7529)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
```

**Comment by hpinhal** [comment]
Hi @ali-star,

Thanks for reporting the exception.

To help us diagnose the problem, it would be great if you could provide some information.

1. Do you have a list of steps to reproduce this consistently?
2. How often does this crash occur? Some numbers would be great.

**Comment by ali-star** [comment]
Hi @hpinhal 
Unfortunately, I couldn't reproduce the crash, I saw it in the firebase console.
We had ~50 crashes in the past 2 weeks.

**Comment by hpinhal** [comment]
Hi @ali-star,

We're adding some safety checks to the surrounding code.
It will be included in patch 3.5.1. You can subscribe to releases on Github to be notified.
